security/pkg/pki/util/crypto.go

```diff
@@ -49,6 +49,36 @@ func ParsePemEncodedCertificate(certBytes []byte) (*x509.Certificate, error) {
 	return cert, nil
 }
 
+// SplitPemEncodedCertificates will break a concatenated PEM file into parts and
+// parse each block as a x509 certificate. Useful for cert chains and root certs.
+// The last element in the list is returned as PEM - in the case of cert chain, it should
+// be the root certificate, as Istio appends the root at the end.
+func SplitPemEncodedCertificates(certBytes []byte) ([]*x509.Certificate, error) {
```

Comment:
waht's the diff with `ParsePemEncodedCertificateChain`

Reply:
added comment - it's the last element that is parsed as well.